x-pack/plugins/synthetics/server/legacy_uptime/lib/saved_objects/migrations/monitors/8.9.0.ts

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { EncryptedSavedObjectsPluginSetup } from '@kbn/encrypted-saved-objects-plugin/server';
+import { SavedObjectUnsanitizedDoc } from '@kbn/core/server';
+import { ConfigKey, SyntheticsMonitorWithSecrets } from '../../../../../../common/runtime_types';
+import { SYNTHETICS_MONITOR_ENCRYPTED_TYPE } from '../../synthetics_monitor';
+
+export const migration890 = (encryptedSavedObjects: EncryptedSavedObjectsPluginSetup) => {
+  return encryptedSavedObjects.createMigration<
+    SyntheticsMonitorWithSecrets,
+    SyntheticsMonitorWithSecrets
+  >({
+    isMigrationNeededPredicate: function shouldBeMigrated(
+      doc
+    ): doc is SavedObjectUnsanitizedDoc<SyntheticsMonitorWithSecrets> {
+      return true;
+    },
+    migration: (
+      doc: SavedObjectUnsanitizedDoc<SyntheticsMonitorWithSecrets>
+    ): SavedObjectUnsanitizedDoc<SyntheticsMonitorWithSecrets> => {
+      let migrated = doc;
+      migrated = {
+        ...migrated,
+        attributes: {
+          ...migrated.attributes,
+          [ConfigKey.ALERT_CONFIG]: migrated.attributes[ConfigKey.ALERT_CONFIG] || {
+            status: {
+              enabled: true,
+            },
+            tls: {
+              enabled: true,
+            },
+          },
```

Comment:
This won't work for `8.8.x` -> `8.9.x` migration as `8.8` already has the `attributes[ConfigKey.ALERT_CONFIG]` object containing `{ status: enabled: true }`.
```suggestion
          [ConfigKey.ALERT_CONFIG]: {
            status: {
              enabled: true,
            },
            tls: {
              enabled: true,
            },
            ...(migrated.attributes[ConfigKey.ALERT_CONFIG] ?? {})
          },
```